### ISSUE TITLE
Develop branch unavailable

### ISSUE THREAD

**Issue by nothingelsematters7** [python-mode/python-mode#624]
After checkouting your reposiroty I don't see develop branch, but I can see it here from web interface

```
» git remote -v                                                                                                              
origin  https://git::@github.com/klen/python-mode.git (fetch)
origin  https://git::@github.com/klen/python-mode.git (push)
```

After doing `git fetch origin` I have following branches

```
» git branch -a -v
* master                4bda303 Merge pull request #561 from vcrini/master
  remotes/origin/master 4bda303 Merge pull request #561 from vcrini/master
```

So may be there are some settings for your repository so I cannot see develop branch?
And why do you have two different branches master and develop? And which should I use?
Thank you in advance
##

**Comment by vlad003** [comment]
I just cloned the repo and I can see the develop branch just fine:

```
$ git branch -a -v
* develop                 fd70ac2 Fix #600
  remotes/origin/HEAD     -> origin/develop
  remotes/origin/develop  fd70ac2 Fix #600
  remotes/origin/gh-pages 85ff12d Update deb
  remotes/origin/master   4bda303 Merge pull request #561 from vcrini/master
```

There are two branches because develop is used for development, and master holds the stable code that gets released. If you look at the release tags and the commit history on the master branch, you'll notice there are merge commits from `release/VERSION` and that merge commit is also tagged with that version.

**Comment by timfeirg** [comment]
close this issue? @nothingelsematters7
